internal/querynodev2/segments/segment_loader.go

```diff
@@ -1332,14 +1302,6 @@ func (loader *segmentLoader) LoadDeltaLogs(ctx context.Context, segment Segment,
 		return err
 	}
 
-	metrics.QueryNodeNumEntities.WithLabelValues(
```

Comment:
should not delete this metric.
length of delta log is not real deleted entry in these segment .